eng/pipelines/runtime-staging.yml

```diff
@@ -64,9 +64,14 @@ jobs:
     buildConfig: Release
     runtimeFlavor: mono
     platforms:
+    - Android_x86
     - Android_x64
+    - Android_arm
```

Comment:
this will need to be moved next to the `Android_arm64` location below since we don't want to run device based tests on PRs right now.

Reply:
actually, maybe in a new section since we only want to build but not run `Android_arm` right now (our devices are all arm64 on Helix)